### ISSUE TITLE
"Force disable newly built LTN Combinators" settings not applying

### ISSUE THREAD

**Issue by JStMorgan** [kryojenik/LTN_Combinator_Modernized#50]
I have a set of LTN blueprints that I'm maintaining via your mod (many thanks for putting the time into it!), but have been getting frustrated by updated blueprints not working as I expected.

**Steps to Reproduce Issue**

1. Have an existing LTN blueprint (see first screenshot).
![20230903220351_1](https://github.com/kryojenik/LTN_Combinator_Modernized/assets/47102583/ed90021a-6aa5-44a3-9a10-82dcda3b381e)
2. Add an entity to the blueprint and update the blueprint and select new contents. Note that the LTN Combinator is not touched in this step.
3. Take your blueprint and place it again.
4. Observe that a non-default Requester threshold has been set, despite not being present in the original and not being set by the user (second screenshot).
![20230903220344_1](https://github.com/kryojenik/LTN_Combinator_Modernized/assets/47102583/d88c0fb3-8044-4835-b321-1262075b5c3c)
5. Change the "Force disable newly built LTN Combinators" to "Powered off", which per the in-game tooltip should affect new, blueprinted and copied combinators. The Disabled Alert setting is toggled On.
6. Repeat steps 1-3.
7. Observe that a non-default Requester threshold is still set, with both Provider and Requester services enabled. No Disabled Alert is received.

This looks to be similar as reported in #29 and #34 .

**Comment by kryojenik** [comment]
Do you have a save game that is demonstrating this with affected blueprints.  I think I'm following the steps you have laid out, but I must be missing a detail or two.  I am unable to reproduce what you're describing.

**Comment by kryojenik** [comment]
Is the blueprint one that was created pre-2.0?  Also, what are the mod-settings you have configured?

**Comment by JStMorgan** [comment]
I've reproduced the issue locally in a new save with a minimum set of mods. The only ones used are LTN, LTN Combinator Modernised (latest version) and Editor Extensions to provide a test environment. The save game and two blueprint strings (a basic provider station before modification, and one after) are attached below. 
In the save game, I have created a basic 1-1 provider station, blueprinted it, added some entities, updated the existing blueprint and then placed that updated blueprint.

[LTN_IssueTesting.zip](https://github.com/kryojenik/LTN_Combinator_Modernized/files/12737145/LTN_IssueTesting.zip)
[Station BP Strings.txt](https://github.com/kryojenik/LTN_Combinator_Modernized/files/12737154/Station.BP.Strings.txt)

The only setting in LTN Combinator Modernised that I have changed is setting `Force disable newly-built LTN Combinators` to `Neither disabled`. The current tool-tip states that "If building from a blueprint, services will keep the state of the blueprint", but I am not seeing this behaviour if an already-existing blueprint is modified in any way.
